SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialDispatcher.h

```diff
@@ -45,6 +71,11 @@ class SPATIALGDK_API USpatialDispatcher : public UObject
 	UPROPERTY()
 	USpatialStaticComponentView* StaticComponentView;
 
-	UPROPERTY()
-	TMap<uint32, UOpCallbackTemplate*> UserOpCallbacks; // key has to be explicit primitive or UHT complains
+	// This index is incremented and returned every time an AddOpCallback function is called.
+	// CallbackIds enable you to deregister callbacks using the RemoveOpCallback function. 
+	// RunCallbacks is called by the SpatialDispatcher and executes all user registered 
+	// callbacks for the matching component ID and network operation type.
+	FCallbackId NextCallbackId;
+	TMap<Worker_ComponentId, TMap<Worker_OpType, TArray<UserOpCallbackData>>> ComponentOpTypeToCallbackMap;
```

Comment:
This TMap type is quite verbose and used in a few places, worth adding a `using` definition